components/com_content/src/Model/CategoryFormModel.php

```diff
@@ -0,0 +1,501 @@
+<?php
+
+/**
+ * @package     Joomla.Site
+ * @subpackage  com_content
+ *
+ * @copyright   (C) 2024 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Component\Content\Site\Model;
+
+use Joomla\CMS\Access\Access;
+use Joomla\CMS\Access\Rules;
+use Joomla\CMS\Factory;
+use Joomla\CMS\Form\Form;
+use Joomla\CMS\Helper\TagsHelper;
+use Joomla\CMS\Language\Multilanguage;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Plugin\PluginHelper;
+use Joomla\Database\ParameterType;
+use Joomla\Registry\Registry;
+use Joomla\Utilities\ArrayHelper;
+use Joomla\Component\Categories\Administrator\Model\CategoryModel as AdminCategoryModel;
+use Joomla\Component\Categories\Administrator\Table\CategoryTable;
+
+// phpcs:disable PSR1.Files.SideEffects
+\defined('_JEXEC') or die;
+// phpcs:enable PSR1.Files.SideEffects
+
+/**
+ * Frontend category form model for com_content.
+ *
+ * @since  6.1.0
+ */
+class CategoryFormModel extends AdminCategoryModel
+{
+    /**
+     * Component option name.
+     *
+     * @var    string
+     * @since  6.1.0
+     */
+    protected $option = 'com_content';
+
+    /**
+     * Model typeAlias string. Used for version history.
+     *
+     * @var    string
+     * @since  6.1.0
```

Comment:
```suggestion
     * @since   __DEPLOY_VERSION__
```

doing this means our build scripts will correctly set the version - it might not make it to 6.1 ;)